python/ray/dashboard/utils.py

```diff
@@ -479,15 +479,23 @@ def __str__(self):
 class NotifyQueue:
     """Asyncio notify queue for Dict signal."""
 
-    _queue = asyncio.Queue()
+    _queue = None
+
+    @classmethod
+    def queue(cls):
```

Comment:
```suggestion
    @classmethod
    @property
    def _queue(cls):
```

In this way other places do not need to be changed.